mteb/abstasks/AbsTaskRetrieval.py

```diff
@@ -109,7 +109,7 @@ class AbsTaskRetrieval(AbsTask):
 
     ignore_identical_ids: bool = False
     abstask_prompt = "Retrieve text based on user query."
-    k_values: list[int] = [1, 3, 5, 10, 20, 100, 1000]
+    k_values: list[int] = [1, 2, 3, 5, 10, 20, 100, 1000]
```

Comment:
I don't want to do this, is there another way?

Reply:
Do you know where/how? I tried adding `k_values` to the metadata, but it failed.

Reply:
Ah, beautiful, thank you! That's a nice way to do it.